src/scss/mixins/_bem.scss

```diff
@@ -96,3 +98,20 @@
     @content;
   }
 }
+
+
+$bootstrap-ratio-mobile: 12 / $grid-columns-mobile;
+$bootstrap-ratio-tablet: 12 / $grid-columns-tablet;
+$bootstrap-ratio-laptop: 12 / $grid-columns-laptop;
+$bootstrap-ratio-desktop: 12 / $grid-columns-desktop;
+
+@mixin bootstrap-span($property, $cols) {
```

Comment:
please move this to it's own module: `scss/mixins/_bootstrap.scss`. This has nothing to do with BEM (BlockElementModifier)